site/docs/reference/Connectors/capture-connectors/http-file.md

```diff
@@ -0,0 +1,136 @@
+---
+sidebar_position: 10
+---
+# HTTP file
+
+This connector captures data from an HTTP endpoint into a Flow collection.
+
+[`ghcr.io/estuary/source-http-file:dev`](https://ghcr.io/estuary/source-http-file:dev) provides the latest connector image. You can also follow the link in your browser to see past image versions.
+
+## Supported data types
+
+This connector automatically captures the data hosted at the specified URL into a single Flow collection.
+
+The following file types are supported:
+
+* Avro
+* CSV
+* JSON
+* W3C Extended Log
+
+The following compression methods are supported:
+
+* ZIP
+* GZIP
+* ZSTD
+
+By default, Flow automatically detects the file type and compression method.
+If necessary, you can specify the correct file type, compression, and other properties (CSV only) using the optional [parser configuration](#advanced-parsing-http-hosted-data).
+
+## Prerequisites
+
+To use this connector, you'll need the URL to an HTTP endpoint that hosts data of one of the supported types.
```

Comment:
One thing that's important is that the endpoint must support [`HEAD`](https://developer.mozilla.org/en-US/docs/web/http/methods/head) HTTP requests which includes a [`Last-Modified`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Last-Modified) header that indicates the last time the document was updated. We use this information to know when to pull the data again.